ts/client/src/client.ts

```diff
@@ -2515,6 +2598,12 @@ export class MangoClient {
     // but user would potentially open new positions.
     banks: Bank[],
     perpMarkets: PerpMarket[],
+    additionalAccounts: AdditionalHealthAccounts = {
```

Comment:
so because remaining accounts need to be in the right order i added option in here to pass it optionally to build function